eng/scripts/Language-Settings.ps1

```diff
@@ -6,11 +6,17 @@ $packagePattern = "*.nupkg"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/dotnet-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=dotnet%2F&delimiter=%2F"
 
-function Get-dotnet-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
+function Get-dotnet-PackageInfoFromRepo ($pkgDirectoryPath, $serviceDirectoryName)
```

Comment:
If we change these argument names and counts then it will be a breaking change so we will need to figure out how to stage these correctly.